DATA_SOURCES.md

```diff
@@ -204,10 +205,11 @@ For many European countries, data is available from [ENTSO-E](https://transparen
 - Bulgaria:
   - Hydro: [NEK](https://nek.bg/index.php/en/our-business/electricity-generation)
   - Other: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
-- Canada (British Columbia, New Brunswick, Newfoundland and Labrador, Nova Scotia, Prince Edward Island):
+- Canada (New Brunswick, Newfoundland and Labrador, Nova Scotia, Prince Edward Island):
   [wikipedia.org](https://en.wikipedia.org/wiki/List_of_generating_stations_in_Canada)
 - Canada (Alberta): [AESO](http://ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet),
   [wikipedia.org](https://en.wikipedia.org/wiki/List_of_generating_stations_in_Alberta)
+- Canada (British Columbia): [Wikipedia.org](https://en.wikipedia.org/wiki/List_of_generating_stations_in_British_Columbia)
```

Comment:
If you change the data source here you also need to make sure it matches the config file for the zone in question, CA-BC, otherwise you need to update that as well here: `config/zones/CA-BC.yaml`